cpp/src/unary/math_ops.cu

```diff
@@ -264,6 +282,7 @@ std::unique_ptr<column> unary_op_with(column_view const& input,
   using Type                     = device_storage_type_t<T>;
   using FixedPointUnaryOpFunctor = FixedPointFunctor<Type>;
 
+  // When scale is >= 0 and unary_op is CEIL or FLOOR, the unary_operation for is a no-op
```

Comment:
"the unary_operation for this is a no-op"? or maybe "the unary_operation is a no-op".